model/riscv_vmem_sv32.sail

```diff
@@ -214,6 +210,9 @@ function translate32(asid, ptb, vAddr, ac, priv, mxr, do_sum, level, ext_ptw) =
                   MemException(e) => internal_error(__FILE__, __LINE__, "invalid physical address in TLB")
                 };
                 TR_Address(ent.pAddr | EXTZ(vAddr & ent.vAddrMask), ext_ptw)
+              } else {
```

Comment:
Inverting these is a little gratuitous, though perhaps it makes more sense

Reply:
Changed back the if/then ordering